src/components/shared/Loading.vue

```diff
@@ -3,22 +3,13 @@
     <img
       src="~@/assets/icons/logo-animated-circle-reduced-clearnest.svg"
       class="loading__icon"
-      v-if="hasCleanestColor"
     />
-    <img src="~@/assets/icons/logo-animated-circle-reduced.svg"
-    class="loading__icon" v-else />
   </div>
 </template>
 
 <script>
 export default {
   name: 'Loading',
-  props: {
```

Comment:
wouldn't it be good to also remove the prop from where the component was called?

Reply:
Done!